### ISSUE TITLE
Cannot determine column separator

### ISSUE THREAD

**Issue by raipc** [uniVocity/univocity-parsers#305]
Column separator detector doesn't determine the correct delimiter if line separator is CRLF and the file contains more than one comment lines without column separator

```java
    @Test
    public void testSeparatorDetectionCrLfAndMultilineComment() {
        String csv = "#created at 2019-01-22T11:39:43.312Z\r\n" +
                "#CSV export\r\n" +
                "Timestamp;Value;Metric;Entity;host\r\n" +
                "2019-01-21T11:39:53.763Z;160527072;jvm_memory_used;dev;LOCALHOST\r\n" +
                "2019-01-21T11:40:08.765Z;1.6270228E+8;jvm_memory_used;dev;LOCALHOST\r\n" +
                "2019-01-21T11:40:23.765Z;454336496;jvm_memory_used;dev;LOCALHOST\r\n";
        final CsvParserSettings settings = new CsvParserSettings();
        settings.setReadInputOnSeparateThread(false);
        settings.setQuoteDetectionEnabled(true);
        settings.setLineSeparatorDetectionEnabled(true);
        settings.setDelimiterDetectionEnabled(true, ',', ';', '\t', '|', ' ');
        final CsvParser csvParser = new CsvParser(settings);
        csvParser.parseAll(new StringReader(csv))
                .forEach(columns -> assertThat(columns.length, is(5)));
        assertThat(csvParser.getDetectedFormat().getDelimiter(), is(';'));
    }
```

**Comment by jbax** [comment]
Thank you for reporting this bug! I've fixed it and released a 2.8.0-SNAPSHOT build which you can already use and test.

Thanks for using our parsers!

**Comment by raipc** [comment]
Wow, it was very fast! Thank you for the great tool!
